tools/deployment/package-helm/templates/database-logs-pv.yaml

```
@@ -0,0 +1,8 @@
+{{- include "clp.createLocalPv" (dict
+  "root" .
+  "name" (printf "%s-database-logs" (include "clp.fullname" .))
+  "component" "database"
+  "capacity" "5Gi"
+  "accessModes" (list "ReadWriteOnce")
+  "hostPath" (printf "%s/database" .Values.clpConfig.logs_directory)
+) }}
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)